### ISSUE TITLE
templateSimpleExpression - add line number

### ISSUE THREAD

**Issue by rrd108** [rrd108/vue-mess-detector#54]

**Comment by David-Pena** [comment]
I have a question @rrd108 

If I want to print a new thing, in this case the Line Number, how can I reproduce a fail test so I can see the real output (as if I was an end user). I don't have that much experience with tests so I'm a bit confused

![image](https://github.com/user-attachments/assets/fe92bddd-2dc5-4e05-8d49-ef80391fc197)

**Comment by rrd108** [comment]
There are multiple ways to do it.

One is to use the debugger. If you use vscode you can press `Shift` + `Ctrl` + `p` to open the command pallet and start typing debug. There will be an option in the dropdown called `Debug: JavaScript Debug Terminal`
If you select it, it will open a new debug terminal.

Now you can add a breakpoint anywhere in the code by clicking before the line number. A red dot will appear.

If you execute `yarn test` in the JS Debug Terminal it will run all the tests, and will stop at the breakpoint. There you can check any js variable.

The other way is to issue `yarn build` in the normal terminal, and then execute `yarn analyze /path/to/any/vuejs/folder`
This will actually run the analyzer on the selected folder, and you will get all console outputs.
In this case the dist folder will be overwritten, please leave it out from the PR.

**Comment by David-Pena** [comment]
I just created a sample component to test a few things:
![image](https://github.com/user-attachments/assets/5d021bd5-f3f4-4358-a4fb-648f5f33ed70)

And then I run `yarn build` and `yarn analyze /src/demo/DemoComponent.vue` but it only shows me this on console
![image](https://github.com/user-attachments/assets/5a89043c-0e10-4ed2-886e-d429fc503f7c)

And in vscode, the `dist/cli.js` file opens by itself and its content is this
![image](https://github.com/user-attachments/assets/84b7579a-a31b-4e9f-bbbd-6d7303fcac9f)

Is there any specific node version I should use? or why I'm not able to visualize the output for my demo component?

**Comment by rrd108** [comment]
@David-Pena try executing `yarn analyze /src/demo`

`Analyze` accepts folders, not files.

**Comment by David-Pena** [comment]
Good morning @rrd108 

I tried that but the result is the same
![image](https://github.com/user-attachments/assets/d2052b76-f0da-4fcb-a339-6ac8f8cee9c3)

**Comment by rrd108** [comment]
That is strange.

Can you please try to run it in a normal terminal and not the js debug terminal? Do you get the same?

**Comment by David-Pena** [comment]
Yes, it happens with other terminals too (I tried bash and cmd).

I created this draft PR for this issue https://github.com/rrd108/vue-mess-detector/pull/58. It's a draft PR because I couldn't verify it as an end user (I couldn't see the actual output the package gives)

Maybe someone else can try running it?

**Comment by rrd108** [comment]
I did some quick reserch and it seems to be some OS level problem.

Anyway I fixed this at a1203bbb509e6bc5765749afae6e8fc9e9908ae1

Your solution was not in line with the other line number solutions, but I used some idea from your code :)

So you can cloce the PR as the issue is already fixed.

**Comment by David-Pena** [comment]
Hi @rrd108 I finally was able to reproduce the logs without any error, it worked with the build step and then executing the command from the docs `npx vue-mess-detector analyze path/to/fle`. 

With `yarn analyze path/to/file` it doesnt work (I'm not sure if its suppose to but with the command from the docs is more than enough!
